WordPress/Classes/Services/ReaderTopicService.m

```diff
@@ -363,6 +363,10 @@ - (ReaderTopic *)siteTopicForPost:(ReaderPost *)post
     topic.topicDescription = post.blogDescription;
     topic.path = path;
 
+    [self.managedObjectContext performBlockAndWait:^{
```

Comment:
Since few lines above this snippet, we're actually running a fetch request (and inserting an object into the context), i think we either need to wrap up the snippet that calls this method, (or) wrap all of its contents within a performBlockAndWait.

OR we could just get rid of the `performBlockAndWait`, and add an... `NSAssert([NSThread isMainThread], @"")`, and pray for @astralbodies not to notice it!